src/predict.rs

```diff
@@ -292,13 +292,13 @@ pub trait Intra: Dim {
         let output_index = r * stride + c;
 
         // Clamp the output to the correct bit depth
-        output[output_index] = this_pred.max(0).min((1_u32 << bd) - 1) as u16;
+        output[output_index] = this_pred as u16;
```

Comment:
For a record: from Q&A in #daala irc, the reason for removing clamping output is to sync with how libaom does here.